extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/ControllerChatHandler.java

```diff
@@ -58,24 +59,25 @@ public ControllerChatHandler(TaskToolbox toolbox, Controller controller)
   }
 
   /**
-   * Used by subtasks to post {@link ClusterByStatisticsSnapshot} for shuffling stages.
+   * Used by subtasks to post {@link PartialKeyStatisticsInformation} for shuffling stages.
    *
-   * See {@link ControllerClient#postKeyStatistics} for the client-side code that calls this API.
+   * See {@link ControllerClient#postPartialKeyStatistics(StageId, int, PartialKeyStatisticsInformation)}
+   * for the client-side code that calls this API.
    */
   @POST
-  @Path("/keyStatistics/{queryId}/{stageNumber}/{workerNumber}")
+  @Path("/partialKeyStatistics/{queryId}/{stageNumber}/{workerNumber}")
```

Comment:
nit : I would change this to : partialKeyStatisticsInformation

Reply:
Also, we would break backward compatibility here so it's worth mentioning that in the release notes.

Reply:
Changed

Reply:
Added a release notes section.